src/digraphs.c

```diff
@@ -701,7 +701,7 @@ static Obj FuncIS_ANTISYMMETRIC_DIGRAPH(Obj self, Obj adj) {
           level++;
           nbs = ELM_PLIST(adj, j);
           stack += 4;
-          stack[0] = INT_INTOBJ(ADDR_OBJ(nbs)[k]);
+          stack[0] = INT_INTOBJ(ELM_LIST(nbs, k));
```

Comment:
I wouldn't have realised that `ADDR_OBJ` wasn't valid here. But I'm glad I know now.